docs/table_design/Data_distribution.md

```diff
@@ -112,51 +214,208 @@ CREATE TABLE site_access(
     pv BIGINT SUM DEFAULT '0'
 )
 AGGREGATE KEY(event_day, site_id, city_code, user_name)
--- Set the partitioning method as Range partitioning.
-PARTITION BY RANGE(event_day) (
+PARTITION BY RANGE(event_day)(
     PARTITION p1 VALUES LESS THAN ("2020-01-31"),
     PARTITION p2 VALUES LESS THAN ("2020-02-29"),
     PARTITION p3 VALUES LESS THAN ("2020-03-31")
 )
--- Set the bucketing method as Hash bucketing and specify the bucketing key.
-DISTRIBUTED BY HASH(event_day, site_id);
+DISTRIBUTED BY HASH(site_id);
+```
+
+**Create multiple partitions at a time**
+
+Multiple partitions can be created at a time at and after table creation. You can specify the start and end time for all the partitions created at a time in `START()` and `END()` and the partition increment value in `EVERY()`. However, note that the range of partitions is right hand half open, which includes the start time but does not include the end time. The naming rule for partitions is the same as that of dynamic partitioning.
+
+- **Partition a table on a date-type column (DATE and DATETIME) at the table creation**
+
+  When the partitioning column is of date type, at table creation, you can use `START()` and `END()` to specify the start date and end date for all the partitions created at a time, and `EVERY(INTERVAL xxx)` to specify the incremental interval between two partitions. Currently the interval granularity supports `HOUR` (since v3.0), `DAY`, `WEEK`, `MONTH`, and `YEAR`.
+
+  In the following example, the date range of all the partitions created at a time starts from 2021-01-01 and ends on 2021-01-04, with an incremental interval of one day:
+
+    ```SQL
+  CREATE TABLE site_access (
+      datekey DATE,
+      site_id INT,
+      city_code SMALLINT,
+      user_name VARCHAR(32),
+      pv BIGINT DEFAULT '0'
+  )
+  ENGINE=olap
+  DUPLICATE KEY(datekey, site_id, city_code, user_name)
+  PARTITION BY RANGE (datekey) (
+      START ("2021-01-01") END ("2021-01-04") EVERY (INTERVAL 1 DAY)
+  )
+  DISTRIBUTED BY HASH(site_id)
+  PROPERTIES ("replication_num" = "3" );
+    ```
+
+  It is equivalent to using the following `PARTITION BY` clause in the CREATE TABLE statement:
+
+    ```SQL
+  PARTITION BY RANGE (datekey) (
+  PARTITION p20210101 VALUES [('2021-01-01'), ('2021-01-02')),
+  PARTITION p20210102 VALUES [('2021-01-02'), ('2021-01-03')),
+  PARTITION p20210103 VALUES [('2021-01-03'), ('2021-01-04'))
+  )
+    ```
+
+- **Partition a table on a date-type column (DATE and DATETIME)  with different date intervals at the table creation**.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/6d09e65a-2964-480e-ae47-8936f71e493c)
remove one redundant space, and remove "the"